packages/backend/dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use an official Node.js image based on Alpine Linux
+FROM node:18-alpine
+
+# Set the working directory in the container
+WORKDIR /
+
+# Copy package.json and package-lock.json files
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Install nodemon globally
+RUN npm install -g nodemon
+
+# Copy the rest of the application code to the working directory
+COPY . .
+
+# Copy the environment variables file
+COPY .env .env
+
+# Expose the port that the app runs on
+EXPOSE 3000
```

Comment:
I would recommend using port from .env:

option1:
#Copy the environment variables file to the working directory
COPY .env .env

#Expose the port defined in the .env file
#Use a shell form to interpret the $PORT variable
EXPOSE ${PORT}

#Command to run the development server
CMD ["npm", "run", "dev"]

option2:
ports:
- "${PORT}:${PORT}"
env_file:
- .env
command: npm run dev